vector/v.univar/main.c

```diff
@@ -688,7 +748,16 @@ void summary(void)
             quartile_perc = (Cvarr.value[qpos_perc]).val.d;
         }
 
-        if (shell_flag->answer) {
+        if (format == JSON) {
+            json_object_set_number(root_object, "first_quartile", quartile_25);
+            json_object_set_number(root_object, "median", median);
+            json_object_set_number(root_object, "third_quartile", quartile_75);
+
+            char percentile_key[24];
+            snprintf(percentile_key, 24, "percentile_%d", perc);
```

Comment:
Update to match the json schema from our discussion.

```json
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "type": "object",
  "properties": {
    "percentiles": {
      "type": "array",
      "items": {
        "type": "object",
        "properties": {
          "percentile": {
            "type": "number",
            "minimum": 0,
            "maximum": 100
          },
          "value": {
            "type": "number"
          }
        },
        "required": ["percentile", "value"],
        "additionalProperties": false
      }
    }
  },
  "required": ["percentiles"],
  "additionalProperties": false
}

```

Reply:
Done.